pkgs/applications/misc/wtf/default.nix

```
@@ -37,5 +38,6 @@ buildGoModule rec {
     license = licenses.mpl20;
     maintainers = with maintainers; [ kalbasit ];
     platforms = platforms.linux ++ platforms.darwin;
+    broken = stdenv.isDarwin
```

```suggestion
    broken = stdenv.isDarwin;
```
Missing trailing semicolon